docs/RELEASE_NOTES.md

```diff
@@ -26,25 +26,15 @@
 
 7. Add Migration guide section which specifies explicitly if there are breaking changes and how to tackle them.
 
```

Comment:
This diff looks wrong to me. Probably merge conflict is not handled well.

Reply:
ea0287825